index 5842caa..400f159
--- a/node_modules/react-server-dom-webpack/cjs/react-server-dom-webpack-plugin.js
+++ b/node_modules/react-server-dom-webpack/cjs/react-server-dom-webpack-plugin.js
@@ -364,7 +364,8 @@ class ReactFlightWebpackPlugin {
                           context,
                           clientRefDep.request,
                           {},
-                          (err3, resolvedPath) => {
+                          (err3, _, resolvedPathObject) => {
+                            const resolvedPath = resolvedPathObject?.path;
                             if (err3 || "string" !== typeof resolvedPath)
                               return filterCb(null, !1);
                             fs.readFile(
